src/connections/allContacts.js

```diff
@@ -104,3 +102,18 @@ export const withContactsMutations = withMutations(client => ({
 }))
 
 export default flow([withContacts, withContactsMutations])
+
+export function mergeContact(contact, attributes) {
+  function concat(arr1 = [], arr2 = []) {
+    return [...arr1, ...arr2]
+  }
+  function customizer(objValue, srcValue, key) {
+    if (Array.isArray(objValue) || Array.isArray(srcValue)) {
+      return concat(objValue, srcValue)
```

Comment:
We'll probably want to prevent duplicates at some point, but I assume this will happen in another PR :slightly_smiling_face:

Reply:
Ohhh, you are so right.
No, I want to add that to this PR.